DuckDuckGo/uk.lproj/Feedback.strings

```diff
@@ -0,0 +1,68 @@
+/* Class = "UIImageView"; accessibilityLabel = "Negative Feedback"; ObjectID = "1RQ-bi-6hU"; */
+"1RQ-bi-6hU.accessibilityLabel" = "Негативний Відгук";
+
+/* Class = "UINavigationItem"; title = "Share Feedback"; ObjectID = "46v-cB-2YV"; */
+"46v-cB-2YV.title" = "Надіслати Відгук";
+
+/* Class = "UIButton"; normalTitle = "Share Feedback"; ObjectID = "7zN-Xh-yYi"; */
+"7zN-Xh-yYi.normalTitle" = "Надіслати Відгук";
+
+/* Class = "UIButton"; normalTitle = "No Thanks! I’m Done"; ObjectID = "A1s-GY-hbq"; */
+"A1s-GY-hbq.normalTitle" = "Ні, дякую! У мене все";
+
+/* Class = "UINavigationItem"; title = "Share Feedback"; ObjectID = "D8A-B7-Dqu"; */
+"D8A-B7-Dqu.title" = "Надіслати Відгук";
+
+/* Class = "UIBarButtonItem"; title = "Back"; ObjectID = "JF8-Ow-zXL"; */
+"JF8-Ow-zXL.title" = "Назад";
+
+/* Class = "UIViewController"; title = "Share Feedback"; ObjectID = "Rvx-dM-etJ"; */
+"Rvx-dM-etJ.title" = "Надіслати Відгук";
+
+/* Class = "UITextField"; placeholder = "Which website is broken?"; ObjectID = "TMj-NM-j3b"; */
+"TMj-NM-j3b.placeholder" = "Який вебсайт має проблеми?";
+
+/* Class = "UINavigationItem"; title = "Report Broken Site"; ObjectID = "WjT-HM-yim"; */
+"WjT-HM-yim.title" = "Повідомити про не справний сайт";
```

Comment:
Based on this dictionary https://slovnyk.ua/index.php?swrd=%D0%BD%D0%B5%D1%81%D0%BF%D1%80%D0%B0%D0%B2%D0%BD%D0%B8%D0%B9 and usage of this word I guess we should update this translation
```suggestion
"WjT-HM-yim.title" = "Повідомити про несправний сайт";
```